tests/components/jellyfin/__init__.py

```diff
@@ -2,9 +2,18 @@
 import json
 from typing import Any
 
+from homeassistant.core import HomeAssistant
+from homeassistant.loader import bind_hass
+
 from tests.common import load_fixture
 
 
 def load_json_fixture(filename: str) -> Any:
     """Load JSON fixture on-demand."""
     return json.loads(load_fixture(f"jellyfin/{filename}"))
+
+
+@bind_hass
```

Comment:
You should be able to drop the `bind_hass` here since its only needed when not passing `hass`